Tools version is unsupported

The [xunit sample](https://github.com/stajs/SpecFlow.NetCore/tree/master/samples/VS2017/SpecFlow%202.1.0/net461/Sample.Website.Tests.XUnit) in this repository does not build spec.cs files on my machine.

Withing the build output there is a call to `specflow.exe generateall` and it generates the following error:

```
The tools version "14.0" is unrecognized. Available tools versions are "2.0", "3.5", "4.0".  D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Sample.Website.Tests.XUnit.csproj.fake.csproj
```
I receive the same error from the developer command line using a manually created 'fake.csproj' file.

Is the issue caused because I only have Visual Studio 2017 on my machine? Does the project build successfully on machines with VS 2015 installed as well?

I'm aware of Issue #49 but that issue was closed because Target Framework is not supported.

Full build log.

```
Restoring NuGet packages...
To prevent NuGet from restoring packages during build, open the Visual Studio Options dialog, click on the Package Manager node and uncheck 'Allow NuGet to download missing packages during build.'
1>------ Rebuild All started: Project: Sample.Website, Configuration: Debug Any CPU ------
1>Sample.Website -> D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website\bin\Debug\net461\win7-x86\Sample.Website.exe
2>------ Rebuild All started: Project: Sample.Website.Tests.XUnit, Configuration: Debug Any CPU ------
2>Found: C:\Users\tim\.nuget\packages\SpecFlow\2.1.0\tools\specflow.exe
2>Current directory: D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit
2>Found: D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Sample.Website.Tests.XUnit.csproj
2>Generating fake csproj.
2><?xml version="1.0" encoding="utf-8"?>
2><Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
2>	<PropertyGroup>
2>		<RootNamespace>SpecFlow.GeneratedTests</RootNamespace>
2>		<AssemblyName>SpecFlow.GeneratedTests</AssemblyName>
2>	</PropertyGroup>
2>	<ItemGroup>
2>		<None Include="app.config">
2>			<SubType>Designer</SubType>
2>		</None>
2>		<None Include="Root.feature">
2>			<Generator>SpecFlowSingleFileGenerator</Generator>
2>			<LastGenOutput>Root.feature.cs</LastGenOutput>
2>		</None>
2>		<None Include="Folder\Folder.feature">
2>			<Generator>SpecFlowSingleFileGenerator</Generator>
2>			<LastGenOutput>Folder.feature.cs</LastGenOutput>
2>		</None>
2>	</ItemGroup>
2></Project>
2>Saving: D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Sample.Website.Tests.XUnit.csproj.fake.csproj
2>Found xunit in D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Sample.Website.Tests.XUnit.csproj
2>Validating app.config.
2>Found xunit in D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Sample.Website.Tests.XUnit.csproj
2>Calling: C:\Users\tim\.nuget\packages\SpecFlow\2.1.0\tools\specflow.exe generateall "D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Sample.Website.Tests.XUnit.csproj.fake.csproj" /force /verbose
2>The tools version "14.0" is unrecognized. Available tools versions are "2.0", "3.5", "4.0".  D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Sample.Website.Tests.XUnit.csproj.fake.csproj
2>
2>Removing: D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Sample.Website.Tests.XUnit.csproj.fake.csproj
2>Fixing SpecFlow generated files
2>New file generated: D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Root.feature.cs. No tests in Root.feature will be discovered by 'dotnet test'
2>New file generated: D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\Folder\Folder.feature.cs. No tests in Folder.feature will be discovered by 'dotnet test'
2>Rebuild to make the above files discoverable, see https://github.com/stajs/SpecFlow.NetCore/issues/22.
2>SpecFlow fixed.
2>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(1987,5): warning MSB3270: There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website\bin\Debug\net461\win7-x86\Sample.Website.exe", "x86". This mismatch may cause runtime failures. Please consider changing the targeted processor architecture of your project through the Configuration Manager so as to align the processor architectures between your project and references, or take a dependency on references with a processor architecture that matches the targeted processor architecture of your project.
2>Sample.Website.Tests.XUnit -> D:\Downloads\SpecFlow.NetCore-master\SpecFlow.NetCore-master\samples\VS2017\SpecFlow 2.1.0\net461\Sample.Website.Tests.XUnit\bin\Debug\net461\Sample.Website.Tests.XUnit.dll
2>Done building project "Sample.Website.Tests.XUnit.csproj".
========== Rebuild All: 2 succeeded, 0 failed, 0 skipped ==========
```

--- a/src/SpecFlow.NetCore/Args.cs
+++ b/src/SpecFlow.NetCore/Args.cs
@@ -10,10 +10,12 @@ namespace Specflow.NetCore
 		public const string SpecFlowPathArgName = "--specflow-path";
 		public const string WorkingDirectoryArgName = "--working-directory";
 		public const string TestFrameworkArgName = "--test-framework";
+		public const string ToolsVersionArgName = "--tools-version";
 
 		public string SpecFlowPath { get; }
 		public DirectoryInfo WorkingDirectory { get; }
 		public string TestFramework { get; }
+		public string ToolsVersion { get; }
 
 		public Args(string[] args)
 		{
@@ -53,6 +55,10 @@ namespace Specflow.NetCore
 
 					case TestFrameworkArgName:
 						TestFramework = value;
+						break;
+
+					case ToolsVersionArgName:
+						ToolsVersion = value;
 						break;
 
 					default:

--- a/src/SpecFlow.NetCore/Fixer.cs
+++ b/src/SpecFlow.NetCore/Fixer.cs
@@ -16,13 +16,13 @@ namespace SpecFlow.NetCore
 		private FileInfo[] _featureFiles;
 		private readonly string _toolsVersion;
 
-		public Fixer(string specFlowPath = null, string testFramework = null, string toolsVersion = null)
+		public Fixer(string specFlowPath = null, string testFramework = null, string toolsVersion = "14.0")
 		{
 			_specFlowExe = FindSpecFlow(specFlowPath);
 			WriteLine("Found: " + _specFlowExe);
 
 			_testFramework = testFramework;
-			_toolsVersion = toolsVersion ?? "14.0";
+			_toolsVersion = toolsVersion;
 		}
 
 		private string FindSpecFlow(string path)

--- a/src/SpecFlow.NetCore/Fixer.cs
+++ b/src/SpecFlow.NetCore/Fixer.cs
@@ -14,13 +14,15 @@ namespace SpecFlow.NetCore
 		private readonly string _specFlowExe;
 		private string _testFramework;
 		private FileInfo[] _featureFiles;
+		private readonly string _toolsVersion;
 
-		public Fixer(string specFlowPath = null, string testFramework = null)
+		public Fixer(string specFlowPath = null, string testFramework = null, string toolsVersion = null)
 		{
 			_specFlowExe = FindSpecFlow(specFlowPath);
 			WriteLine("Found: " + _specFlowExe);
 
 			_testFramework = testFramework;
+			_toolsVersion = toolsVersion ?? "14.0";
 		}
 
 		private string FindSpecFlow(string path)
@@ -184,8 +186,8 @@ namespace SpecFlow.NetCore
 			var sb = new StringBuilder();
 
 			// Set the "ToolsVersion" to VS2013, see: https://github.com/techtalk/SpecFlow/issues/471
-			sb.Append(@"<?xml version=""1.0"" encoding=""utf-8""?>
-<Project ToolsVersion=""14.0"" DefaultTargets=""Build"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
+			sb.Append($@"<?xml version=""1.0"" encoding=""utf-8""?>
+<Project ToolsVersion=""{_toolsVersion}"" DefaultTargets=""Build"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
 	<PropertyGroup>
 		<RootNamespace>SpecFlow.GeneratedTests</RootNamespace>
 		<AssemblyName>SpecFlow.GeneratedTests</AssemblyName>

--- a/src/SpecFlow.NetCore/Program.cs
+++ b/src/SpecFlow.NetCore/Program.cs
@@ -10,7 +10,7 @@ namespace SpecFlow.NetCore
 			try
 			{
 				var a = new Args(args);
-				var fixer = new Fixer(a.SpecFlowPath, a.TestFramework);
+				var fixer = new Fixer(a.SpecFlowPath, a.TestFramework, a.ToolsVersion);
 				fixer.Fix(a.WorkingDirectory);
 
 				PrintUsingColor("SpecFlow fixed.", ConsoleColor.Green);

